stemcell_builder/stages/bosh_ntpd/apply.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+set -e
+
+base_dir=$(readlink -nf $(dirname $0)/../..)
+source $base_dir/lib/prelude_apply.bash
+source $base_dir/lib/prelude_bosh.bash
+
+mkdir -p $chroot/$bosh_dir/src
+
+# ntpd
+ntpd_basename=ntp-4.2.8p4
```

Comment:
any reason to use ntpd packaged as an asset instead of installing it from an appropriate package manager?